sdk/aqueduct/client.py

```diff
@@ -223,6 +223,9 @@ def create_param(self, name: str, default: Any, description: str = "") -> BaseAr
         check_explicit_param_name(self._dag, name)
         if isinstance(default, LocalData):
             default = extract_val_from_local_data(default)
+            artifact = create_param_artifact(self._dag, name, default, description)
+            artifact._from_local_data = True
```

Comment:
Following the advice above, add a `from_local_data` argument to `create_param_artifact()`.